src/dialogs/PaymentErrorDialog.vue

```diff
@@ -103,5 +103,21 @@ export default class PaymentErrorDialog extends Vue {
   get numWarnings (): number {
     return this.warnings?.length || 0
   }
+
+  /** The formated error message */
+  protected formatedError = []
+  protected mounted (): void {
+    this.formatedError = this.errors.map((item:any) => (
+      item.message.split('<br/>').map((item2:string, index2:number) => {
+        return index2 === 0 ? item2 + '<br/>' : '<span>' + item2 + '</span><br/>'
+      }).join('')
+    ))
+  }
```

Comment:
Transform the local variable into the array of error that can use the css style.

Reply:
Don't do this. The error message should be a single block of text and should be displayed as such.

Did you look at my suggestion to use flexbox and 2 "columns" inside it, the first for the bullet and the second for the text?

Reply:
The error message is a single text. I cannot turn columns in just the text with {{ error.messasge }}.

Reply:
Use v-html.

Use https://vuetifyjs.com/en/styles/flex/.

Reply:
![image](https://user-images.githubusercontent.com/51458187/164810006-3fef2e69-e46e-4634-bfad-423294f2d118.png)

It cannot align.

Reply:
Doesn't it align if you use flexbox and create a flex item for the bullet and another flex item for the message?

Reply:
i've done something like this in create ui in the past.  You would just need to update the `<v-col cols="11">` to use `v-html`.  I think it indented the bullet text properly.  But I'm assuming it can be simplified to use fewer tags.  Something like what Severin is suggesting of just using flex box.  

https://github.com/bcgov/business-create-ui/blob/81b41f9ce19ab16af1a1d3764c57adc1e318e6bb/src/components/Dissolution/CompleteResolution.vue#L274-L282

Reply:
The problem is that there is no html element to wrap around each text for the v-html span tag.

Reply:
![image](https://user-images.githubusercontent.com/51458187/164823893-12793217-94a9-4624-b916-80850eb9b6a2.png)

This is what v-html gives. I don't know how I can do css style to plain text for which there is no html element.

Reply:
@Lcyc29 Your code in Vue SFC Playground doesn't have Vuetify available to it, so you won't have the Vuetify flex helper classes either.

Reply:
@argush3 Thanks Argus. I ended up using something similar to yours, but your code `<v-icon>mdi-circle-small</v-icon>` helped me a lot with the icon.

Reply:
This is code is removed.